### PR TITLE
v2v: add windows2022 to matrix of auto testing

### PR DESCRIPTION
**Pull request by vwu-vera**
Add the latest windows version: win2022 into auto testing.

### CODE REVIEW

File: v2v/tests/cfg/convert_vm_to_libvirt.cfg [hunk 1]
```diff
@@ -114,6 +114,9 @@
                 - win2019:
                     only esx
                     os_version = "win2019"
+                - win2022:
+                    only source_esx.esx_65,  source_esx.esx_67, source_esx.esx_70
```
Comment (xiaodwan) [inline v2v/tests/cfg/convert_vm_to_libvirt.cfg:118]: 'only esx' is fine.
We don't care whether the guests are in esx server or not in the code. It's users responsibility to make sure
the guests are exist when running cases.

File: v2v/tests/cfg/convert_vm_to_ovirt.cfg [hunk 2]
```diff
@@ -146,6 +146,10 @@
                 - win2019:
                     no xen
                     os_version = "win2019"
+                - win2022:
+                    no xen
+                    only source_esx.esx_65,  source_esx.esx_67, source_esx.esx_70
```
Comment (xiaodwan) [inline v2v/tests/cfg/convert_vm_to_ovirt.cfg:151]: Delete this line, it's controlled in 'esx' part.

**Comment by vwu-vera** [comment]
@xiaodwan Please check the modification according to the suggestion. Thanks.

**Review by kamilvarga — APPROVED** [review]
LGTM.

**Review by xiaodwan — APPROVED** [review]

**Pull request merged**